docs/source/markdown/podman-kube-play.1.md

```diff
@@ -30,6 +30,9 @@ Note: If the `:latest` tag is used, Podman will attempt to pull the image from a
 
 Note: The command `podman play kube` is an alias of `podman kube play`, and will perform the same function.
 
+Note: The command `podman kube down` can be used to stop and remove pods or containers based on the same Kubernetes YAML used
```

Comment:
Note: The command `podman kube down` can be used to stop and remove pods or containers based on the same Kubernetes YAML created by the `podman kube play`  command.